app/views/static_pages/home.html.erb

```diff
@@ -1,6 +1,12 @@
-  <h1><%= (t".title")%></h1>
-  <p>
-    <%= (t".text1")%>
-    <%= link_to (t ".link_name"), (t ".link")%> <%= (t".text2")%>
+<div class="center jumbotron">
+  <h1><%= t("text") %></h1>
 
+  <h2>
+    <%= (t".text1") %>
```

Comment:
Sử dụng ngoặc lộn xộn quá đó chế nhé

Sau `t` ko có ngoặc thì phải thêm space
ko thì bỏ ngoặc vào

self check lại toàn bộ html (dead)